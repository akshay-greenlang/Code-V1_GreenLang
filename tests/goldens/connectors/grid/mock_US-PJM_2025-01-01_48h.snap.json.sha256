564465610afaaeaa528c7cd670943e7e2686747c39ac4439691431e4adb56ac2