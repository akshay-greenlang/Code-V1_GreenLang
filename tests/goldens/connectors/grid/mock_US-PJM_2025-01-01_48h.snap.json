{"connector_id":"grid/intensity/mock","connector_version":"0.1.0","payload":{"metadata":{"algorithm":"seasonal_sinusoid","connector":"grid/intensity/mock","data_points":48,"region_offset":84,"seed":"f6cf61b113f86517b0a403264a89bb053dbd2a2d5cc54f00512c7d46ffaf7fdc","version":"0.1.0"},"region":"US-PJM","resolution":"hour","series":[{"quality":"simulated","ts":"2025-01-01T00:00:00Z","unit":"gCO2/kWh","value":"384"},{"quality":"simulated","ts":"2025-01-01T01:00:00Z","unit":"gCO2/kWh","value":"435.8"},{"quality":"simulated","ts":"2025-01-01T02:00:00Z","unit":"gCO2/kWh","value":"484"},{"quality":"simulated","ts":"2025-01-01T03:00:00Z","unit":"gCO2/kWh","value":"525.4"},{"quality":"simulated","ts":"2025-01-01T04:00:00Z","unit":"gCO2/kWh","value":"557.2"},{"quality":"simulated","ts":"2025-01-01T05:00:00Z","unit":"gCO2/kWh","value":"577.2"},{"quality":"simulated","ts":"2025-01-01T06:00:00Z","unit":"gCO2/kWh","value":"584"},{"quality":"simulated","ts":"2025-01-01T07:00:00Z","unit":"gCO2/kWh","value":"577.2"},{"quality":"simulated","ts":"2025-01-01T08:00:00Z","unit":"gCO2/kWh","value":"557.2"},{"quality":"simulated","ts":"2025-01-01T09:00:00Z","unit":"gCO2/kWh","value":"525.4"},{"quality":"simulated","ts":"2025-01-01T10:00:00Z","unit":"gCO2/kWh","value":"484"},{"quality":"simulated","ts":"2025-01-01T11:00:00Z","unit":"gCO2/kWh","value":"435.8"},{"quality":"simulated","ts":"2025-01-01T12:00:00Z","unit":"gCO2/kWh","value":"384"},{"quality":"simulated","ts":"2025-01-01T13:00:00Z","unit":"gCO2/kWh","value":"332.2"},{"quality":"simulated","ts":"2025-01-01T14:00:00Z","unit":"gCO2/kWh","value":"284"},{"quality":"simulated","ts":"2025-01-01T15:00:00Z","unit":"gCO2/kWh","value":"242.6"},{"quality":"simulated","ts":"2025-01-01T16:00:00Z","unit":"gCO2/kWh","value":"210.8"},{"quality":"simulated","ts":"2025-01-01T17:00:00Z","unit":"gCO2/kWh","value":"190.8"},{"quality":"simulated","ts":"2025-01-01T18:00:00Z","unit":"gCO2/kWh","value":"184"},{"quality":"simulated","ts":"2025-01-01T19:00:00Z","unit":"gCO2/kWh","value":"190.8"},{"quality":"simulated","ts":"2025-01-01T20:00:00Z","unit":"gCO2/kWh","value":"210.8"},{"quality":"simulated","ts":"2025-01-01T21:00:00Z","unit":"gCO2/kWh","value":"242.6"},{"quality":"simulated","ts":"2025-01-01T22:00:00Z","unit":"gCO2/kWh","value":"284"},{"quality":"simulated","ts":"2025-01-01T23:00:00Z","unit":"gCO2/kWh","value":"332.2"},{"quality":"simulated","ts":"2025-01-02T00:00:00Z","unit":"gCO2/kWh","value":"384"},{"quality":"simulated","ts":"2025-01-02T01:00:00Z","unit":"gCO2/kWh","value":"435.8"},{"quality":"simulated","ts":"2025-01-02T02:00:00Z","unit":"gCO2/kWh","value":"484"},{"quality":"simulated","ts":"2025-01-02T03:00:00Z","unit":"gCO2/kWh","value":"525.4"},{"quality":"simulated","ts":"2025-01-02T04:00:00Z","unit":"gCO2/kWh","value":"557.2"},{"quality":"simulated","ts":"2025-01-02T05:00:00Z","unit":"gCO2/kWh","value":"577.2"},{"quality":"simulated","ts":"2025-01-02T06:00:00Z","unit":"gCO2/kWh","value":"584"},{"quality":"simulated","ts":"2025-01-02T07:00:00Z","unit":"gCO2/kWh","value":"577.2"},{"quality":"simulated","ts":"2025-01-02T08:00:00Z","unit":"gCO2/kWh","value":"557.2"},{"quality":"simulated","ts":"2025-01-02T09:00:00Z","unit":"gCO2/kWh","value":"525.4"},{"quality":"simulated","ts":"2025-01-02T10:00:00Z","unit":"gCO2/kWh","value":"484"},{"quality":"simulated","ts":"2025-01-02T11:00:00Z","unit":"gCO2/kWh","value":"435.8"},{"quality":"simulated","ts":"2025-01-02T12:00:00Z","unit":"gCO2/kWh","value":"384"},{"quality":"simulated","ts":"2025-01-02T13:00:00Z","unit":"gCO2/kWh","value":"332.2"},{"quality":"simulated","ts":"2025-01-02T14:00:00Z","unit":"gCO2/kWh","value":"284"},{"quality":"simulated","ts":"2025-01-02T15:00:00Z","unit":"gCO2/kWh","value":"242.6"},{"quality":"simulated","ts":"2025-01-02T16:00:00Z","unit":"gCO2/kWh","value":"210.8"},{"quality":"simulated","ts":"2025-01-02T17:00:00Z","unit":"gCO2/kWh","value":"190.8"},{"quality":"simulated","ts":"2025-01-02T18:00:00Z","unit":"gCO2/kWh","value":"184"},{"quality":"simulated","ts":"2025-01-02T19:00:00Z","unit":"gCO2/kWh","value":"190.8"},{"quality":"simulated","ts":"2025-01-02T20:00:00Z","unit":"gCO2/kWh","value":"210.8"},{"quality":"simulated","ts":"2025-01-02T21:00:00Z","unit":"gCO2/kWh","value":"242.6"},{"quality":"simulated","ts":"2025-01-02T22:00:00Z","unit":"gCO2/kWh","value":"284"},{"quality":"simulated","ts":"2025-01-02T23:00:00Z","unit":"gCO2/kWh","value":"332.2"}],"unit":"gCO2/kWh"},"provenance":{"connector_id":"grid/intensity/mock","connector_version":"0.1.0","metadata":{"data_points":48,"end":"2025-01-03T00:00:00+00:00","region":"US-PJM","resolution":"hour","start":"2025-01-01T00:00:00+00:00"},"mode":"record","query_hash":"1df0febf80b4599bc8a26e8596d932d0aa383502ced83a0cdd8d2e2c41bce12e","schema_hash":"2cf663a8a03d50acb0c1df2fba40940ffb4f63f0d1882a6386a70320e2bc06f6","seed":"f6cf61b113f86517b0a403264a89bb053dbd2a2d5cc54f00512c7d46ffaf7fdc","snapshot_id":"c193c0c35ee2e658e6b52a37c02f91306357444f07204be2efcd67b50f96eecf","timestamp":"2025-10-09T05:13:40.767920Z"},"schema":{"payload_model":"GridIntensityPayload","provenance_model":"ConnectorProvenance"}}